presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -461,13 +456,8 @@ public void commitInsert(ConnectorSession session, ConnectorInsertTableHandle in
         List<ColumnInfo> columns = handle.getColumnHandles().stream().map(ColumnInfo::fromHandle).collect(toList());
 
         shardManager.commitShards(transactionId, tableId, columns, parseFragments(fragments), externalBatchId);
-    }
 
-    @Override
-    public void rollbackInsert(ConnectorSession session, ConnectorInsertTableHandle insertHandle)
-    {
-        RaptorInsertTableHandle handle = checkType(insertHandle, RaptorInsertTableHandle.class, "insertHandle");
-        shardManager.rollbackTransaction(handle.getTransactionId());
+        clearRollback();
```

Comment:
Yes, we should rollback so that transaction gets makeed as aborted immediately and the created shards can be cleaned up.

Is that not what happens with these changes?